tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/backfill_lifestyle_test.py

```diff
@@ -0,0 +1,133 @@
+"""
+Integration test for BackfillLifestyle.
+"""
+# Python Imports
+import os
+
+# Third party imports
+from dateutil import parser
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.backfill_lifestyle import BackfillLifestyle
+from common import JINJA_ENV, OBSERVATION, PERSON
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+OBSERVATION_TMPL = JINJA_ENV.from_string("""
+INSERT INTO `{{project}}.{{dataset}}.observation`
+(observation_id, person_id, observation_concept_id, observation_date, observation_type_concept_id, observation_source_concept_id)
```

Comment:
You could add the `questionnaire_response_id` field here if you feel it is appropriate.